UiPath/Script.Activities
# Merge Localization from Language team

Description
Fixes for localization issues of packages
How to test
Change Studio language and check translations for activities packages
Jira
Label
